valheim_plus.cfg

```diff
@@ -66,6 +66,18 @@ noWeatherDamage=false
 ; The maximum range that you can place build objects at
 maximumPlacementDistance=5
 
+; When destroying a building piece, setting this to true will ensure it always drops full resources.
+alwaysDropResources=false
+
+; When destroying a building piece, setting this to true will ensure it always drops pieces that the devs have marked as "do not drop".
+alwaysDropExcludedResources=false
+
+; Setting this to true will cause repairing with the hammer to repair in a radius.
+enableAreaRepair=false
+
+; Tweaks the area repair radius
```

Comment:
Could we be more specific here? Maybe something like "Sets the area repair radius. A value of 7.5 would mean your repair radius is 7.5m. Requires `enableAreaRepair=true`"